lib/main.js

```diff
@@ -223,35 +223,30 @@ if (options.interactiveTests) {
     })
   })
 
-  ava.test('graphics: Kernel boot logo', async (test) => {
-    return utils.runManualTestCase(test, {
-      prepare: [ 'Plug a monitor in the device\'s HDMI output' ],
-      do: [ 'Reboot the device' ],
-      assert: [ 'The Tux (Linux) logo should not be visible on the screen' ]
+  if (deviceTypeContract.hdmi) {
+    ava.test('graphics: Kernel boot logo/Reboot splash screen', async (test) => {
+      return utils.runManualTestCase(test, {
+        prepare: [ 'Plug a monitor in the device\'s HDMI output' ],
+        do: [ 'Reboot the device' ],
+        assert: [
+          'The Resin.io logo splash screen should be visible when the board initiates reboot',
+          'The Tux (Linux) logo should not be visible on the screen while device is booting',
+          'The Resin.io logo splash screen should be visible during boot-up'
+        ]
+      })
     })
-  })
 
-  ava.test('graphics: Reboot splash screen', async (test) => {
-    return utils.runManualTestCase(test, {
-      prepare: [ 'Plug a monitor in the device\'s HDMI output' ],
-      do: [ 'Reboot the device' ],
-      assert: [
-        'The Resin.io logo splash screen should be visible when the board initiates reboot',
-        'The Resin.io logo splash screen should be visible during boot-up'
-      ]
-    })
-  })
-
-  ava.test('graphics: Boot/Shutdown splash screen', async (test) => {
-    return utils.runManualTestCase(test, {
-      prepare: [ 'Plug a monitor in the device\'s HDMI output' ],
-      do: [ 'Reboot the device' ],
-      assert: [
-        'The Resin.io logo splash screen should be visible when the board initiates shutdown',
-        'The Resin.io logo splash screen should be visible during boot-up'
-      ]
+    ava.test('graphics: Boot/Shutdown splash screen', async (test) => {
+      return utils.runManualTestCase(test, {
+        prepare: [ 'Plug a monitor in the device\'s HDMI output' ],
+        do: [ 'Shutdown the device' ],
+        assert: [
+          'The Resin.io logo splash screen should be visible when the board initiates shutdown',
+          'The Resin.io logo splash screen should be visible during boot-up'
```

Comment:
This implies the user is powering on the device after shutting it down. Should we add this to the `do` section?